### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by Utwo**
Refactor all project a little.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,59 +1,67 @@
...
+    return res.status(500).json({ message: "Failed in fetching UIDL" });
   }
-})
+});
 
-app.use('/', router);
-app.listen(port, () => {
-  console.log('Server running succesfully')
-})
+app.get("/", async (req, res) => res.send("REPL API server"));
```
Comment (JayaKrishnaNamburu) [inline src/index.ts]: I think, ``async`` is not required here

**Review by JayaKrishnaNamburu — APPROVED** [review]
Everything looks good 👍

**Comment by JayaKrishnaNamburu** [comment]
And why ``yarn.lock`` file is removed 🤔

**Comment by Utwo** [comment]
> And why `yarn.lock` file is removed thinking

yarn.lock it's still there

**Comment by JayaKrishnaNamburu** [comment]
> > And why `yarn.lock` file is removed thinking
> 
> yarn.lock it's still there

Yeah, overlooked 😅 sorry 😁👍

**Pull request merged**